ployst/github/management/commands/github_poke.py

```diff
@@ -37,12 +38,19 @@ def handle(self, *args, **options):
             'repository': {
                 'url': repo_url
             },
-            'ref': branch_name
+            'ref': branch_name,
+            'head_commit': {
+                'id': '1234567'
+            }
         })
-        sig = compute_github_signature(load, org, repo)
+        secret = get_secret(org, repo)
+        sig = compute_github_signature(load, secret)
 
         response = requests.post(full_url, data=load,
-                                 headers={'X_HUB_SIGNATURE': sig})
```

Comment:
Hrm no I think this is purely `requests` code. The header was not coming through. I don't think django was filtering it out, although it might have been.